controllers/tasks.js

```diff
@@ -321,12 +321,28 @@ const updateTaskStatus = async (req, res, next) => {
     if (task.taskData.status === TASK_STATUS.VERIFIED || req.body.status === TASK_STATUS.MERGED)
       return res.boom.forbidden("Status cannot be updated. Please contact admin.");
 
+    if (userStatusFlag) {
+      if (task.taskData.status === TASK_STATUS.DONE && req.body.percentCompleted < 100) {
+        if (req.body.status === TASK_STATUS.DONE || !req.body.status) {
+          return res.boom.badRequest("Task percentCompleted can't updated as status is DONE");
+        }
+      }
+
```

Comment:
Why is task status DONE being checked in 2 if conditions?

Reply:
Hi @prakashchoudhary07 sorry for late reply...my health is not good from last 3 days
Reason found from vinayak-g who written this code
<img width="699" alt="image" src="https://github.com/Real-Dev-Squad/website-backend/assets/107163260/6e286680-8dc9-40c4-94c2-1121e03ffa98">
https://github.com/Real-Dev-Squad/website-backend/pull/725#discussion_r998593197